sbom/spdx/encoder.go

```diff
@@ -0,0 +1,444 @@
+package spdx
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"runtime/debug"
+	"slices"
+	"strconv"
+	"strings"
+	"time"
+
+	spdxjson "github.com/spdx/tools-golang/json"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/sbom"
+
+	"github.com/spdx/tools-golang/spdx/common"
+	v2common "github.com/spdx/tools-golang/spdx/v2/common"
+	"github.com/spdx/tools-golang/spdx/v2/v2_3"
+)
+
+type Version string
```

Comment:
won't block the PR, but I think each public field usually has a godoc in this repo